sympy/physics/mechanics/functions.py

```diff
@@ -532,7 +539,7 @@ def _recurser(expr, sub_dict):
 
 def _fraction_decomp(expr):
     """Return num, den such that expr = num/den"""
-    if not isinstance(expr, Mul):
+    if not isinstance(expr, MulClass):
```

Comment:
For this PR, this should be fixed so that `sympy.core.backend` provides exactly the same names as sympy does, e.g. `from symengine import Tan as tan`.

In general, we will hit the problem that some small things might be different (perhaps some method, or such). Then I think we need to use this solution:

https://github.com/symengine/symengine/wiki/SymPy-core-upgrade-to-SymEngine